test_utils/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+Test utilities.
+"""
+# Since py.test discourages putting __init__.py into test directory (i.e. making tests a package)
+# one cannot import from anywhere under tests folder. However, some utility classes/methods might be useful
+# in multiple test modules. So this package the place to put them.
```

Comment:
This seems like a good idea to me -- should we consider adding it to the cookiecutter spec?